starport/templates/app/launchpad/Dockerfile.plush

```diff
@@ -0,0 +1,14 @@
+# Compile
+FROM golang:alpine AS builder
+WORKDIR /src/app/
+COPY go.mod go.sum* ./
+RUN go mod download
+COPY . .
+RUN for bin in cmd/*; do CGO_ENABLED=0 go build -o=/usr/local/bin/$(basename $bin) ./cmd/$(basename $bin); done
```

Comment:
It is not crucial right now but in future it may make sense to use `startport build` here and for Stargate.

In future `config.yml` might hold some info about how the app should be built -which will be interpreted by `starport build`.

Reply:
oh nice!  I had thought about that, as well.  

II will open an issue on it and get that into the next batch.